tests/Capgemini.PowerApps.PackageDeployerTemplate.IntegrationTests/CapgeminiPackageTemplateTests.cs

```diff
@@ -122,5 +122,18 @@ public void CapgeminiPackageTemplate_SdkStepConfiguredToDeactivate_SdkStepIsDeac
 
             workflow["statecode"].As<OptionSetValue>().Value.Should().Be(1);
         }
+
+        [Theory]
+        [InlineData("Account Creation Trigger -> Terminate", 0)]
+        [InlineData("Account Creation Trigger1 -> Terminate", 1)]
+        public void CapgeminiPackageTemplate_FlowsAreActivated(string workflowName, int stateCode)
+        {
+            var subjectQuery = new QueryByAttribute("workflow");
```

Comment:
I think `workflowQuery` is more appropriate?

Reply:
Its the workflow name, or am I missing something.

Reply:
Think it's in reference to this @markcunninghamuk -

```suggestion
            var workflowQuery = new QueryByAttribute("workflow");
```

Reply:
Ah I got you, thanks!